compute/gpu/reference-content/choosing-gpu-instance-type.mdx

```diff
@@ -0,0 +1,46 @@
+---
+meta:
+  title: Choosing the right GPU Instance type
+  description: This page provides information about how to choose a GPU Instance type
+content:
+  h1: Choosing the right GPU Instance type
+  paragraph: This section provides information about how to choose a GPU Instance type
+tags: NVIDIA GPU cloud instance
+dates:
+  validation: 2023-08-31
+  posted: 2022-08-31
+categories:
+  - compute
+---
+
+A GPU Instance refers to a virtual computing environment provided by Scaleway that offers access to powerful Graphics Processing Units (GPUs) over the internet.
+GPUs are specialized hardware originally designed for rendering graphics in video games and other 3D applications.
+However, their massively parallel architecture makes them well-suited for various high-performance computing tasks, such as machine learning, data processing, scientific simulations, and more.
+
+Scaleway GPU Instances' availability has revolutionized how researchers, developers, and organizations train complex machine-learning models faster and more efficiently.
+It empowers European AI startups, giving them the means (without the need for a huge CAPEX investment) to create products that revolutionize how we work and live.
+
+## How to choose the right GPU Instance type
+
+Scaleway provides a range of GPU Instance offers, there are several factors to consider when choosing the right GPU Instance type, to ensure that it meets your performance, budget, and scalability requirements.
+Below, you will find a guide to help you make an informed decision:
+
+* **Workload requirements:** Identify the nature of your workload. Are you running machine learning, deep learning, high-performance computing (HPC), data analytics, or graphics-intensive applications? Different Instance types are optimized for different types of workloads. For example, the H100 is not designed for graphics rendering. However, the L4 and L40S are. As [stated by Tim Dettmers](https://timdettmers.com/2023/01/30/which-gpu-for-deep-learning/),  “Tensor Cores are most important, followed by memory bandwidth of a GPU, the cache hierarchy, and only then FLOPS of a GPU.”. For more information, refer to the [NVIDIA GPU portfolio](https://www.nvidia.com/content/dam/en-zz/solutions/data-center/data-center-gpu-portfolio-line-card.pdf).
```

Comment:
For the quote, I think maybe there is context missing - is he talking about deep learning? If so, maybe it should say ....As stated by Tim Dettmers, when it comes to deep learning, tensor cores are most important followed by bla bla "